File: Task-03/Go/Sub-Task3.go
package main

import (
    "fmt"
    "strconv"
)
func main() {
    var input string
    fmt.Print("Enter the number of rows: ")
    fmt.Scanln(&input)
    rows, err := strconv.Atoi(input)
    if err != nil || rows <= 0 {
        fmt.Println("Please enter a valid positive number.")
        return
    }
    for i := 0; i < rows; i++ {
        for j := 0; j < rows-i-1; j++ {
            fmt.Print(" ")
        }
        for j := 0; j <= i; j++ {
            fmt.Print("* ")
        }
        fmt.Println()
    }
    for i := 0; i < rows-1; i++ {
        for j := 0; j <= i; j++ {
            fmt.Print(" ")
        }
        for j := 0; j < rows-i-1; j++ {
            fmt.Print("* ")
        }
        fmt.Println()
    }
}

File: Task-03/Rust/Sub-Task3.rs
use std::io;
fn main() {
    let mut input = String::new();
    println!("Enter the number of rows: ");
    io::stdin().read_line(&mut input).expect("Failed to read line");

    let rows: usize = match input.trim().parse() {
        Ok(num) => num,
        Err(_) => {
            eprintln!("Please enter a valid number");
            return;
        }
    };
    for i in 0..rows {
        for j in 0..(rows - i - 1) {
            print!(" ");
        }
        for j in 0..(i + 1) {
            print!("* ");
        }
        println!();
    }
    for i in 0..(rows - 1) {
        for j in 0..(i + 1) {
            print!(" ");
        }
        for j in 0..(rows - i - 1) {
            print!("* ");
        }
        println!();
    }
}
